scripts/requirements-ci.txt

```diff
@@ -1,90 +1,108 @@
+aenum==3.0.0
 alabaster==0.7.12
 anytree==2.8.0
-appdirs==1.4.3
+appdirs==1.4.4
 arrow==0.17.0
-astroid==2.4.1
-attrs==19.3.0
-Babel==2.8.0
-breathe==4.14.2
+astroid==2.4.2
+attrs==20.3.0
+Babel==2.9.0
+breathe==4.26.1
```

Comment:
Since the documentation now builds on GH, maybe you could consider removing all documentation specific packages. The versions look good though!

Reply:
thx,  I removed requirements-doc.txt from the input list which generates this file in requirements-ci.sh so they will be removed next release.